bigdata-client/src/main/java/com/bigdata/rdf/sail/remote/BigdataSailRemoteRepositoryConnection.java

```diff
@@ -797,7 +798,17 @@ public void setDataset(Dataset arg0) {
             public boolean getIncludeInferred() {
                throw new UnsupportedOperationException();
             }
-   
+
+				@Override
+				public void setMaxExecutionTime(int i) {
```

Comment:
How can these be empty methods? Suggest chasing the open RDF docs

Reply:
The docs say (see https://bitbucket.org/openrdf/sesame/src/71eb79608cd97696598fce5cf85bfd8aad641e74/core/query/src/main/java/org/openrdf/query/Operation.java?at=sesame-2.8.11&fileviewer=file-view-default#Operation.java-97):

```
	 * @param maxQueryTime
	 *        The maximum query time, measured in seconds. A negative or zero
	 *        value indicates an unlimited execution time (which is the default).
	 * @since 2.8.0
```
So by default it's unlimited, and I am not sure we can do anything here - as this class does not have any means to enforce execution time. Any ideas what can be done here instead?

Reply:
My point is that overriding these methods removes the ability to control the maximum execution time through their API.  Why would you do that?

Reply:
But to implement the interface I need to provide some implementation for these methods. Should it just remember the parameter for setMaxExecutionTime() without doing anything? Because it doesn't look like there's implementation of any logic involving it anywhere.